n64.ld

```diff
@@ -22,22 +22,24 @@ OUTPUT_ARCH (mips)
 EXTERN (_start)
 ENTRY (_start)
 
+__mem_start = 0x80000000;
```

Comment:
I'm not 100% won on this new symbol ... I think in a way we hide in clarity because if I see `mem_start` in C code I might be not sure what it means (it's not a standard symbol, so it's something libdragon specific I have too lookup, and it's even defined in .ld so it's also non-trivial to find). On the other hand, `0x80000000` is very clear to whoever should be able to touch and read this code.

Reply:
I totally agree and still I couldn't find a better way. Let me try and think about hardcoded alternative once more...